widgets/eventsFilter/src/EventFilter.tsx

```diff
@@ -82,8 +97,9 @@ function EventFilter({ toolbox }) {
         debouncedContextUpdate(toolbox, updatedFields);
     }
 
-    function handleOptionAddition(e, { name, value }) {
-        setOptions({ ...options, [name]: [{ text: value, value }, ...options[name]] });
+    function handleOptionAddition(_event: React.SyntheticEvent<HTMLElement, Event>, data: DropdownProps) {
```

Comment:
We can simplify this code piece, by doing

```suggestion
    const handleOptionAddition: DropdownProps['onAddItem'] = (_event, data) => {
```

Reply:
done - https://github.com/cloudify-cosmo/cloudify-stage/pull/2446/commits/9e6dce8fa54ba7c5ba49d9b2c893ff32174a7677 ✔️